Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/architecture/Component.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+namespace saintNecEngine
```

Comment:
Question de noob: Tout se met dans le namespace ?

Reply:
since this project will be quite big, using namespaces can be usefull to separate code